src/dataLoaderService.ts

```diff
@@ -53,6 +53,15 @@ export class DataLoaderService {
     return this.loaders[type] as DataLoader<K, V>;
   }
 
+  get userSettings() {
+    return this.getLoader<{ userId: string }, Settings>({
+      type: 'userSettings',
+      loadFn: async ({ userId }) =>
+        this.ctx.con.getRepository(Settings).findOneBy({ userId }),
```

Comment:
should we return `null` if `userId`is undefined for any reason, just thinking if mistake happens it is going to query all user settings because of typeorm..

Reply:
```suggestion
        if (!userId) {
            return null
        }

        this.ctx.con.getRepository(Settings).findOneBy({ userId }),
```

Reply:
Returning null if no userId